proxy/HostStatus.h

```diff
@@ -86,6 +86,7 @@ struct HostStatus {
   void setHostStatus(const char *name, const HostStatus_t status, const unsigned int down_time, const char *reason);
   HostStatus_t getHostStatus(const char *name);
   void createHostStat(const char *name);
+  void loadStats();
```

Comment:
Should we call this function loadHostStatusFromStats() or something to indicate that this function is trying to load the Host Status from stats?